src/config.ts

```diff
@@ -1,21 +1,21 @@
 /**
  * This file contains configuration for most aspects of the website that might need frequent changing.
- * 
+ *
```

Comment:
nit: is there an eslint rule for this? i'm not seeing any warnings for trailing whitespaces

Reply:
if there is, let's just add it to `package.json`'s eslint rules

Reply:
My VS Code settings auto trims whitespace. I'll add the eslint rule: https://eslint.org/docs/rules/no-trailing-spaces.